tensorflow/compiler/xla/service/llvm_ir/llvm_target_helper.h

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Unclear to me whether we need a separate file just for this enum?

Reply:
I have added xla/gpu/target_util.cc and  xla/gpu/target_util.h